### PR TITLE
React validated input component - brighitte johnson

### PR DESCRIPTION
**Pull request by brij8**
react-validated-input-component

### CODE REVIEW

File: react-validated-input-component/src/validated-input.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    if (event.target.value.length < 8) {
+      $err.innerHTML = 'Password must be 8 characters.';
+      $icon.innerHTML = '❌';
+    } else if (event.target.value.length === 0) {
+      $err.innerHTML = 'A password is required.';
+      $icon.innerHTML = '❌';
+    } else {
+      $err.innerHTML = '';
+      $icon.innerHTML = '✅';
+    }
```
Comment (scbowler) [inline react-validated-input-component/src/validated-input.jsx:23]: We never want to do direct DOM manipulation with React. A better approach would be to have these checks in the `render` method, set the values to a variable, then use those variables in your JSX to determine what is rendered.

**Review by scbowler — CHANGES_REQUESTED** [review]
Good job getting it to work, but a few updates needed to avoid direct DOM manipulation. Let me know if you have any questions.

File: react-validated-input-component/src/validated-input.jsx (hunk 1 above)
Comment (brij8) [reply react-validated-input-component/src/validated-input.jsx]: awesome thank you, i knew there was something...

looks much better now

**Review by scbowler — APPROVED** [review]
Much better! Great job! 👍

**Pull request merged**